packages/maskbook/src/extension/options-page/DashboardDialogs/About.tsx

```diff
@@ -72,15 +72,19 @@ const useStyles = makeStyles((theme) =>
 export function DashboardAboutDialog(props: WrappedDialogProps) {
     const { t } = useI18N()
     const classes = useStyles()
-    const { version } = globalThis.browser?.runtime.getManifest() ?? {}
+    const version = globalThis.browser?.runtime.getManifest()?.version ?? process.env.TAG_NAME.slice(1)
     return (
         <DashboardDialogCore {...props} CloseIconProps={{ className: classes.close }}>
             <section className={classes.wrapper}>
                 <header className={classes.header}>
                     <Avatar className={classes.maskface} src="/MB--CircleCanvas--WhiteOverBlue.svg"></Avatar>
                     <img className={classes.masktext} src="/maskbook-title-white.svg" />
                     <Typography className={classes.version} variant="body2" color="inherit">
-                        {t('version_of_stable', { version })}
+                        {t(process.env.build === 'stable' ? 'version_of_stable' : 'version_of_unstable', {
```

Comment:
Can our i18n analyze tool analyze this pattern? @septs

Reply:
https://github.com/DimensionDev/Maskbook/blob/c7e25c5ead7c13c3d301d4eb028d589b91143859/scripts/locale-kit.ts#L90-L92

Reply:
Those part have been moved to #1875.